test/auth/features/oidc/broker.go

```diff
@@ -151,38 +186,41 @@ func BrokerSendEventWithOIDCTokenToReply() *feature.Feature {
 	// Install subscriber
 	f.Setup("install subscriber", eventshub.Install(subscriber,
 		eventshub.ReplyWithTransformedEvent(replyEventType, replyEventSource, ""),
-		eventshub.StartReceiver))
+		eventshub.StartReceiverTLS))
 
 	// Install sink for reply
 	// Hint: we don't need to require OIDC auth at the reply sink, because the
 	// actual reply is sent to the broker ingress, which must support OIDC. This
 	// reply sink is only to check that the reply as sent and routed correctly.
 	f.Setup("install sink for reply", eventshub.Install(reply,
-		eventshub.StartReceiver))
+		eventshub.StartReceiverTLS))
 
 	// Install broker
 	f.Setup("install broker", broker.Install(brokerName, broker.WithEnvConfig()...))
 	f.Setup("Broker is ready", broker.IsReady(brokerName))
+	f.Setup("Broker has HTTPS address", broker.ValidateAddress(brokerName, addressable.AssertHTTPSAddress))
```

Comment:
Not sure if we need this, as we have `f.Prerequisite("transport encryption is strict", featureflags.TransportEncryptionStrict())`. IMO this should be tested in a TLS related test :shrug: